src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1840,6 +1840,7 @@ services:
     arguments:
       - '@prestashop.adapter.form.choice_provider.feature_values_choice_provider'
       - '@PrestaShopBundle\Form\FormCloner'
+      - '@translator'
```

Comment:
Seems like it's an error (maybe related to the rebase) but the listener doesn't use translator in its constructor